packages/server-ct/src/project-ct.js

```diff
@@ -109,7 +109,7 @@ class Project {
     .then((modifiedCfg) => {
       debug('plugin config yielded: %o', modifiedCfg)
 
-      const updatedConfig = config.updateWithPluginValues(cfg, modifiedCfg)
+      const updatedConfig = config.updateWithPluginValues(cfg, modifiedCfg, ['browsers'])
```

Comment:
Why is it necessary to skip validating the `browsers` key?

Reply:
Agreed, I might prefer defaulting browsers to empty for now or defaulting to chrome. We may eventually use this key.

Reply:
This check is not useful here because this key is never used in component testing (see runner-ct.js).

https://github.com/cypress-io/cypress/blob/8e08fbbce2d5ebd9a5b8c6c2033d9825097b7c36/packages/runner-ct/lib/runner-ct.js#L27-L36

It defaults to an empty array already. It works which is 

I am still struggling to find where the browser is actually launched in the codebase.

Reply:
I just found it in server-ct and it is hardcoded. HUM. will probably fix that then.

https://github.com/cypress-io/cypress/blob/f38d121e04117268a7df5954493faa20ca2aaab3/packages/server-ct/index.js#L23-L32

Reply:
@JessicaSachs @lmiller1990 what do you think of it now ?